processor-steps/src/main/java/boringyuri/processor/common/steps/UriFactoryGeneratorStep.kt

```diff
@@ -198,23 +212,23 @@ class UriFactoryGeneratorStep internal constructor(
     }
 
     private fun obtainQueryParams(
-        methodParameters: List<VariableElement>,
-        parameterSpecs: Map<VariableElement, ParameterSpec>
+        methodParameters: List<XVariableElement>,
+        parameterSpecs: Map<XVariableElement, ParameterSpec>
     ): List<QueryParameter> {
         return methodParameters.mapNotNull { param ->
             val paramAnnotation = param.getAnnotation<Param>() ?: return@mapNotNull null
 
             val spec = parameterSpecs.getValue(param)
             val nullable = annotationHandler.isNullable(spec.type, param)
-            val defaultValue = param.getAnnotation<DefaultValue>()?.value
+            val defaultValue = param.getAnnotation<DefaultValue>()
```

Comment:
nit: you may keep this like as it was
```suggestion
            val defaultValue = param.getAnnotation<DefaultValue>()?.value
```